### ISSUE TITLE
Unable to import and use the project in Android Studio

### ISSUE THREAD

**Issue by cprcrack** [naver/android-pull-to-refresh#2]
I'm unable to import and use the project inside my project in Android Studio. What is the easiest way to use it?

**Comment by serj-lotutovici** [comment]
The simplest way would be to copy the library folder and add a build.gradle script which will explain to gradle what to do.

It will take some time to move the hole project to gradle. I'll get to it, and will replay here as soon as I have something.

**Comment by ncoolz** [comment]
Thanks for your comment, serj-lotutovici. I think the library should support aar package as soon. Please wait for next version.

**Comment by ncoolz** [comment]
The library supports now aar package. 

You can use that by adding dependency in build.gradle.

```
 repositories {
    mavenCentral()
}

dependencies {
    compile 'com.navercorp.pulltorefresh:library:3.2.0@aar'
}
```

**Comment by cprcrack** [comment]
Awesome. Will try to import it in the following days and post hopefully successful results here.

**Comment by cprcrack** [comment]
Tested. Works flawlessly.

**Comment by ncoolz** [comment]
Hello, cho.
Please create new github issue and give me a sample project that has the problem. Thanks.

**Comment by markthien** [comment]
hi,
I am using Android Studio 0.8.14 and I have put 

com.navercorp.pulltorefresh:library:3.2.3

inside dependencies section as follow:

dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    compile files('libs/google-http-client-1.16.0-rc.jar')
    compile 'com.github.castorflex.smoothprogressbar:library-circular:1.0.1'
    compile 'com.navercorp.pulltorefresh:library:3.2.3@arr'
}

however, I keep getting:

Error:
Failed to find: com.navercorp.pulltorefresh:library:3.2.3
Open File
Open in Project Structure dialog

Please give me a hand here. Thanks in advance
Mark

**Comment by ncoolz** [comment]
'~:library:3.2.3@**arr**' in dependencies block should be corrected to '~:library:3.2.3@**aar**'.

**Comment by markthien** [comment]
you are awesome man. that the most hilarious mistake I ever make. thanks a lot dude !
